### PR TITLE
Finalize v2016.0

### PR DESCRIPTION
**Pull request by dneto0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,6 +198,16 @@ Use option `-h` to print help.
 The output includes syntax colouring when printing to the standard output stream,
 on Linux, Windows, and OS X.
 
+### Optimizer tool
+
+The optimizer processes a SPIR-V binary module, applying transformations
+in specified order.
```
Comment (antiagainst) [inline README.md]: in the specified order.

File: CHANGES [hunk 2]
```diff
@@ -16,6 +16,8 @@ v2016.0-dev 2016-04-22
      - Supports floating point widths of 16, 32, 64 bits
      - Supports integer widths up to 64 bits
    - Validator is incomplete
+     - Validates some control flow graph rules
```
Comment (antiagainst) [inline CHANGES]: Also ID checks?

File: README.md [hunk 3]
```diff
@@ -198,6 +198,16 @@ Use option `-h` to print help.
...
+
+This is a work in progress, with initially only few available transformations.
```
Comment (antiagainst) [inline README.md]: Prefix with `Warning` like the validator?

Comment (dneto0) [inline README.md]: I left off a Warning because I think it's a different level of expectation.

**Comment by dneto0** [comment]
Addressed feedback

**Comment by antiagainst** [comment]
LGTM.

Let's submit this today so the time stamp is actually correct. ;)

**Pull request merged**